frontend/__tests__/src/utils/Util.test.tsx

```diff
@@ -52,3 +58,36 @@ describe('getJiraToken function', () => {
     expect(jiraToken).toBe('')
   })
 })
+
+describe('findCaseInsensitiveType function', () => {
+  it('Should return as expected when passing in a type that gives case insensitive input bUildkite', () => {
```

Comment:
1. follow team's test case pattern, given when then
2. what is `as exected`? PLZ update

Reply:
Ok, will update later

Reply:
done